pennylane/pauli/pauli_utils.py

```diff
@@ -12,18 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """
-Utility functions used in Pauli partitioning and measurement reduction schemes utilizing the
-symplectic vector-space representation of Pauli words. For information on the symplectic binary
-representation of Pauli words and applications, see:
-
-* `arXiv:quant-ph/9705052 <https://arxiv.org/abs/quant-ph/9705052>`_
-* `arXiv:1701.08213 <https://arxiv.org/abs/1701.08213>`_
-* `arXiv:1907.09386 <https://arxiv.org/abs/1907.09386>`_
+Utility functions used properties of Pauli operations, and converting between
```

Comment:
This file was originally a general `utils.py`. It's been split into `grouping_utils.py` and `pauli_utils.py`.